packages/vite/src/node/plugins/index.ts

```diff
@@ -46,13 +46,15 @@ export async function resolvePlugins(
     ? await (await import('../build')).resolveBuildPlugins(config)
     : { pre: [], post: [] }
   const { modulePreload } = config.build
-  const enableNativePlugin = config.experimental.enableNativePlugin
+  const enableNativePlugin =
```

Comment:
```suggestion
  const enableNativePluginV1 =
```
I think it is confusing to name this `enableNativePlugin`